core/client/db/src/utils.rs

```diff
@@ -32,11 +34,12 @@ use runtime_primitives::traits::{
 	Block as BlockT, Header as HeaderT, Zero, UniqueSaturatedFrom,
 	UniqueSaturatedInto, CheckedConversion
 };
+#[cfg(feature = "kvdb-rocksdb")]
 use crate::DatabaseSettings;
 
 /// Number of columns in the db. Must be the same for both full && light dbs.
 /// Otherwise RocksDb will fail to open database && check its type.
-pub const NUM_COLUMNS: u32 = 9;
+pub const NUM_COLUMNS: u32 = 11;
```

Comment:
What's the migration policy? I haven't seen any so far, is it safe to change the DB?

Reply:
Shouldn't this be 10? In this case I'm not sure if opening a DB with extra columns will break (we should try it). Otherwise, I'd say we ignore the migration since we've broken compatibility in the past with other changes as well.